apps/ejabberd/test/mongoose_rdbms_SUITE.erl

```diff
@@ -86,10 +85,10 @@ keepalive_exit(Config) ->
 meck_config(Server, KeepaliveInterval) ->
     meck:new(ejabberd_config, [no_link]),
     meck:expect(ejabberd_config, get_local_option,
-                fun({odbc_keepalive_interval, _Host}) -> KeepaliveInterval;
-                   ({odbc_start_interval, _Host}) -> 30;
+                fun({odbc_keepalive_interval, odbc_pool, default}) -> KeepaliveInterval;
```

Comment:
Your suggestion makes sense. I don't feel it's necessary, but it would improve the readability and unit test isolation indeed.